src/sentry/static/sentry/app/utils/routeTitle.jsx

```diff
@@ -0,0 +1,8 @@
+const routeTitleGen = (routeName, orgSlug, withSentry = true) => {
```

Comment:
We're trying to add new code as typescript. Could this module be converted to typescript?

Reply:
@markstory didn't know that, i will convert it to ts